template/src/app/pages/Details/Details.tsx

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
-import {Text, StyleSheet} from 'react-native';
-import {SafeAreaView} from 'react-native-safe-area-context';
+import {Text, StyleSheet, View} from 'react-native';
 import Icon from 'react-native-vector-icons/FontAwesome';
 
 export const Details = (): JSX.Element => (
-  <SafeAreaView style={styles.background}>
+  <View style={styles.background}>
```

Comment:
And then here the duplication is removed. This all makes sense when viewed together (I usually review diff by diff, but I guess you get to hear me think out loud then :sweat_smile: )